packages/repl/prebake.mjs

```diff
@@ -21,6 +21,9 @@ export async function prebake() {
   );
   // load samples
   const ds = 'https://raw.githubusercontent.com/felixroos/dough-samples/main/';
+
+  // TODO: move this onto the strudel repo
+  const ts = 'https://raw.githubusercontent.com/todepond/samples/main/';
```

Comment:
https://github.com/TodePond/samples/blob/main/tidal-drum-machines-alias.json

Reply:
it very likely be available on https://strudel.cc/tidal-drum-machines-alias.json after merge/deploy

Reply:
in theory, both github links wouldn't need to be fetched, instead the json could be used directly hmmm